### DISCUSSION TITLE
DHCP Leases Filtering and Search

### DISCUSSION THREAD

**Discussion by Ric878** [TechnitiumSoftware/DnsServer#906] [Ideas]
Just dropping by to suggest some features for DHCP. Apologies in advance if this has already been suggested. In the leases tab, it would be great if you could filter by scope, reserved or not reserved, and perform searches based on IP and/or hostname. That would make dealing with the list much more manageable, especially when troubleshooting.

BTW, love the product!

**Comment by ShreyasZare** [comment]
Thanks for the suggestion. Its already planned for the DHCP server revamp.
